src/test/java/one/util/streamex/MoreCollectorsTest.java

```diff
@@ -591,6 +595,242 @@ public void testToEnumSet() {
         checkCollectorEmpty("Empty", EnumSet.noneOf(TimeUnit.class), MoreCollectors.toEnumSet(TimeUnit.class));
     }
 
+    /**
+     * See {@link MoreCollectors#entriesToMap()}.
+     */
+    @Test
+    public void testEntriesToMap() {
+        assertThrows(IllegalStateException.class, () ->
+                EntryStream.generate(() -> "a", () -> 1).limit(10).collect(MoreCollectors.entriesToMap()));
+
+        checkCollectorEmpty("entriesToMap", Collections.emptyMap(), MoreCollectors.entriesToMap());
+        {
+            Map<CharSequence, CharSequence> expected = Collections.<String, String>emptyMap().entrySet().stream()
+                    .collect(MoreCollectors.entriesToMap());
+            assertTrue(expected.isEmpty());
+        }
+
+        {
+            Map<Integer, String> expected = EntryStream.of(1, "*", 2, "**", 3, "***", 4, "****", 5, "*****").toMap();
+            Supplier<Stream<Entry<Integer, String>>> stream = expected.entrySet()::stream;
+            checkCollector("entriesToMap", expected, stream, MoreCollectors.entriesToMap());
+
+            streamEx(() -> EntryStream.of("one", "*", "two", "**", "three", "***", "four", "****").stream(), supplier -> {
+                Map<CharSequence, CharSequence> result = supplier.get().collect(MoreCollectors.entriesToMap());
+                assertEquals("*", result.get("one"));
+                assertEquals("**", result.get("two"));
+                assertEquals("***", result.get("three"));
+                assertEquals("****", result.get("four"));
+            });
+        }
+    }
+
+    /**
+     * See {@link MoreCollectors#entriesToMap(BinaryOperator)}.
+     */
+    @Test
+    public void testEntriesToMapWithCombiner() {
+        BinaryOperator<String> nullCombiner = null;
+        assertThrows(NullPointerException.class, () -> EntryStream.of(1, "*").stream().collect(MoreCollectors.entriesToMap(nullCombiner)));
```

Comment:
Same: `.stream()` is unnecessary.
`MoreCollectors.entriesToMap(nullCombiner)` must throw NPE before collecting the stream, so the test should look like `assertThrows(NullPointerException.class, () -> MoreCollectors.entriesToMap(nullCombiner))`.